Services/source/housekeeping/housekeeping_service.c

```diff
@@ -21,13 +21,17 @@
 #include <FreeRTOS.h>
 #include <os_task.h>
 #include <os_semphr.h>
+#include <os_projdefs.h>
+#include <os_portmacro.h>
 #include <csp/csp.h>
 #include <csp/csp_endian.h>
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
 #include <time.h>
 
+#include "printf.h"
```

Comment:
Should this be here?

Reply:
Doesn't need to be. I'm done testing with printf for now